tests/tests.py

```diff
@@ -181,6 +187,21 @@ def test_empty_strings_not_allowed(self):
         with self.assertRaises(IntegrityError):
             MinimalTestingModel.objects.create()
 
+    def test_decode_error(self):
+        def mock_decode_error(*args, **kwargs):
+            raise Exception()
+
+        model = MinimalTestingModel.objects.create(pickle_field={'foo': 'bar'})
+        model.save()
+
+        self.assertEqual(
+            {'foo': 'bar'}, MinimalTestingModel.objects.get(pk=model.pk).pickle_field
+        )
+
+        with patch('picklefield.fields.dbsafe_decode', mock_decode_error):
+            encoded_value = dbsafe_encode({'foo': 'bar'})
+            self.assertEqual(encoded_value, MinimalTestingModel.objects.get(pk=model.pk).pickle_field)
+
```

Comment:
What does this patch have to do with Django 2.2 support?

Reply:
Nothing, just coverage improvement :smile: